CONTRIBUTING.md

```diff
@@ -546,3 +546,34 @@ Then, to import it from a file, you'll need to import from the `node_modules` fo
 ```bash
 import * as library_name from "../node_modules/library_name";
 ```
+
+#HTML
+
+Bootstrap divides page into divs with unique ids. Example: content-left. Place html file that will be used into team-software-project/frontend/html. When adding html to screen use:
+
+```javascript
+document.getElementById('div-name').innerHTML = generated-html;
```

Comment:
When you say 'div-name' here does that mean whoever's reading this would change that to 'content' or 'content-left' etc.?

Reply:
Will I change it to something else?

Reply:
I think it would be better to have it in italics and with the wording like: _div id_ . The spacing should help us realise that it's placeholder text and not code.

I'll leave it up to you though, sure you have a higher rank in the team than me :-D. Maybe it's just me who didn't realise that was placeholder text :-P

Reply:
I don't think I can put italics inside the javascript. The markdown didn't recognize the **'s. You might know markdown better than I do if you can figure it out change away!